java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -7009,4 +7010,110 @@ public void lcsIdx(BaseClient client) {
                         () -> client.lcsIdxWithMatchLen(nonStringKey, key1, 10L).get());
         assertInstanceOf(RequestException.class, executionException.getCause());
     }
+
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void sscan(BaseClient client) {
+        String key1 = "{key}-1" + UUID.randomUUID();
+        String key2 = "{key}-2" + UUID.randomUUID();
+        long initialCursor = 0;
+        long defaultCount = 10;
+        String[] numberMembers = new String[125];
+        for (int i = 0; i < numberMembers.length; i++) {
+            numberMembers[i] = String.valueOf(i);
+        }
+        Set numberMembersSet = Set.of(numberMembers);
+        String[] charMembers = new String[] {"a", "b", "c", "d", "e"};
+        Set charMemberSet = Set.of(charMembers);
+        int resultCursorIndex = 0;
+        int resultCollectionIndex = 1;
+
+        // Empty set
+        Object[] result = client.sscan(key1, initialCursor).get();
+        assertEquals(String.valueOf(initialCursor), result[resultCursorIndex]);
+        assertDeepEquals(new String[] {}, result[resultCollectionIndex]);
+
+        // Negative cursor
+        result = client.sscan(key1, -1).get();
+        assertEquals(String.valueOf(initialCursor), result[resultCursorIndex]);
+        assertDeepEquals(new String[] {}, result[resultCollectionIndex]);
+
+        // Result contains the whole set
+        assertEquals(charMembers.length, client.sadd(key1, charMembers).get());
+        result = client.sscan(key1, initialCursor).get();
+        assertEquals(String.valueOf(initialCursor), result[resultCursorIndex]);
+        assertEquals(charMembers.length, ((Object[]) result[resultCollectionIndex]).length);
+        Arrays.stream((Object[]) result[resultCollectionIndex])
+                .forEach(member -> assertTrue(charMemberSet.contains(member)));
```

Comment:
This don't validate that `result[resultCollectionIndex]` has no missing elements. Convert it to a set and then compare entire set with `charMemberSet`.

Reply:
Done.